src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/NomisData.kt

```diff
@@ -500,6 +535,19 @@ interface NomisDataDsl {
     keepBiometrics: Boolean = false,
     dsl: PersonDsl.() -> Unit = {},
   ): Person
+
+  fun corporate(
```

Comment:
There's no DSL marker annotation here - but there is in the implementation class. Not sure which is right or if it matters?